src/workflows-app/utils/submission-utils.ts

```diff
@@ -45,6 +46,26 @@ export type OutputTableData = {
   variable: string;
 };
 
+export type WorkflowMetadata = {
+  actualWorkflowLanguage: string;
+  actualWorkflowLanguageVersion: string;
+  calls: {};
+  end: string;
+  id: string;
+  inputs: [];
+  labels: {};
+  outputs: [];
+  start: string;
+  status: string;
+  submission: string;
+  submittedFiles: {};
+  workflowCallback: {};
+  workflowLog: string;
+  workflowName: string;
+  workflowProcessingEvents: Array<Object>;
```

Comment:
Nit: For consistency with other fields:

```suggestion
  workflowProcessingEvents: {}[];
```